### ISSUE TITLE
Render road ends with fixme=continue differently

### ISSUE THREAD

**Issue by jidanni** [gravitystorm/openstreetmap-carto#4375]
Some future features that extend for an unspecified distance
https://github.com/openstreetmap/iD/issues/8461
will also need a way to be rendered.

**Comment by jidanni** [comment]
I like U+2026 HORIZONTAL ELLIPSIS
`============…`
Not likely to be mistaken for a 'very short trail'. Good!

**Comment by HolgerJeromin** [comment]
First we need an established tag with many uses and than we can decide to render.
Here we only have a vague idea.
Not helpful in this repository.

**Comment by pnorman** [comment]
@jidanni What tag are you proposing is rendered? What is its usage like?

**Comment by jidanni** [comment]
https://github.com/openstreetmap/iD/issues/8461#issuecomment-820802405 mansions with people currently use. However that won't help for map rendering. So we need a way to let people who are looking at the map know that road continues a little further. The use cases are also in the very first comment for that above issue.

**Comment by HolgerJeromin** [comment]
Rendering `fixme` and `todo` tags would render the map unusable for most users.
Not guessing here, I have configured OsmAnd to show them. This gets really ugly.

Thats why this is against the design goal of the style (_Legibility and clarity_):
https://github.com/gravitystorm/openstreetmap-carto/blob/master/CARTOGRAPHY.md#main-goals

**Comment by jidanni** [comment]
> Rendering `fixme` and `todo` tags would render the map unusable for most users.
> Not guessing here, I have configured OsmAnd to show them. This gets really ugly.

Yup. That's why a new tag(?) is needed. As "the path goes on, and doesn't dead end here" would be something that folks would often want to express on the map.

**Comment by pnorman** [comment]
> First we need an established tag with many uses and than we can decide to render.
> Here we only have a vague idea.
> Not helpful in this repository.

Since we still don't have a tag that is being requested to render, there's nothing to do here yet.

**Comment by RobJN** [comment]
I'm not sure why there is a suggestion for a new tag. What would it look like if we render the existing tags (fixme=stub and fixme=continue)? Possibly starting with just rendering if on a node.

P.s. rendering all fixme tags would be overkill but that's not what is being asked here.

**Comment by polarbearing** [comment]
We just declined rendering Fixmes in #4348.
I'd prefer to show fixmes in a way they can be switched off, like the Notes layer, or the developer's switch in OsmAnd, or in editors.

**Comment by HolgerJeromin** [comment]
Perhaps I derailed the discussion into a generic `fixme=*` issue. Sorry for that.

I am pretty sure if someone has a good idea for rendering `fixme=continue` (112000 elements) this issue can be reconsidered. 
Fading the way (which is perhaps what the ASCII art should show) is probably not working.

**Comment by jidanni** [comment]
Well the understanding of people who made Fixmes is that they would not be rendered to the general public.
Therefore I was thinking of a new tag, that from the start, mappers know will be rendered.
Else... well it's like "The Lord came down from the sky, and put everybody underwear on the outside of their clothes."

**Comment by HolgerJeromin** [comment]
An pure tag for painting something on the map should be rejected for good reasons. See also:
https://www.openstreetmap.org/user/imagico/diary/43957
https://www.openstreetmap.org/user/imagico/diary/47432

**Comment by RobJN** [comment]
> Well the understanding of people who made Fixmes is that they would not be rendered to the general public.

That's not my understanding. As I understand it the default rendering is for helping mappers improve the map so I assume that rendering could change to better highlight issues.

**Comment by aceman444** [comment]
> > Well the understanding of people who made Fixmes is that they would not be rendered to the general public.
> 
> That's not my understanding. As I understand it the default rendering is for helping mappers improve the map so I assume that rendering could change to better highlight issues.

The standard osm.org map is often used embedded on ton of other webpages to show location of some shop or facility. Those other sites would not like to display some "debugging" map showing fixmes. The main osm.org style is supposed to give feedback to mappers but only up to some level of detail, to still look professional. There are many other specialized sites showing OSM data in "debugging" mode showing fixmes and even autodetecting and highlighting possible problems.

**Comment by RobJN** [comment]
I think it is possible to render fixme=continue in a way that is useful for mappers and also respectful of those using the default style on other sites.

It's not actually something I'm that bothered about. I started posting here because I was confused why a new tag was being asked for. Some 15 messages in and it feels like we need a proposed style if we are to proceed. 

To try to move us forward: Given that these tags tend to be on the end node, do we need a end node marker or is the render clever enough to associate the end node with the way (and therefore allow the fade out option)?

**Comment by jidanni** [comment]
Sounds great. (Except all the other fixme's aren't rendered, and when they were created the creator never dreamed they would be rendered. Therefore consider a fresh tag (name.))

**Comment by imagico** [comment]
> I think it is possible to render fixme=continue in a way that is useful for mappers and also respectful of those using the default style on other sites.

I would not be opposed to that idea provided a suitable (subtle yet intuitive) way to do that can be found that works for most/all road types.

It had occurred to me before that we render road ends with a round line cap - which, while technically the logical thing to do and it would not be easy to do this differently - it is not at all that this is the obvious choice cartographically.  Most traditional maps do this differently.  It seems perfectly possible to introduce a variant here that implies continuation.

**Comment by imagico** [comment]
Reopening since we have a concrete tag suggested to be rendered now - changed issue title accordingly.
